AUTHORS.txt

```diff
@@ -0,0 +1,5 @@
+The Spyder Kernels Contributors are composed of:
+
+* Carlos Cordoba <ccordoba12@gmail.com> (Spyder-Kernels creator and maintainer)
+* All other developers that have contributed to the spyder-kernels repository:
```

Comment:
This is wrong. It needs to be

    All other developers that have contributed to the Spyder repository

because I didn't take renames when I got the git history of the files took from the Spyder repo (my mistake, but I realized too late).

For example, Pierre is not here, but he was the one that created most of the files present here.

Reply:
...

First I don't push my changes explicitly since

> I hadn't yet addressed your other review comment :-)

Then, since you commented

> But it seems you forgot to push your local changes :-)

I push them with the above disclaimer

and now I'm told

> This is still missing. Please address it.

I just can't win, can I jeje ¯\_(ツ)_/¯

Reply:
TL;DR: I'm still researching the commit history on the relevant files to see what can be done.

Reply:
@ccordoba12 Tis done!